sections/semantics-forms.include

```diff
@@ -1209,7 +1209,7 @@
   <pre highlight="html">
 &lt;form&gt;
   &lt;table&gt;
-    &lt;caption&gt;Example &lt;label&gt;'s for attribute with &lt;label&gt;&lt;/caption&gt;
+    &lt;caption&gt;Example, &lt;label&gt;'s for attribute&lt;/caption&gt;
```

Comment:
@agcolom it's great to comment like this.

I did mean a colon, but grammatical nits in the text of code examples are not that important, so I'll merge this and wait for someone else if they want to fix it further…